Source/FactionColonies/FactionColonies.cs

```diff
@@ -358,7 +358,7 @@ public static void updateChanges()
 
 				str += "\n\nWant to see the full patch notes? Join us on Discord! https://discord.gg/f3zFQqA";
 
-				factionFC.updateVersion = 0.329;
+				factionFC.updateVersion = 0.333;

```

Comment:
At present, the source code is for version 0.333.  The source code in this PR needs to be updated to match the included dll before this PR is merged.